inc/init.php

```diff
@@ -229,8 +229,10 @@
 
 require_once __DIR__.'/src/bootstrap.php';
 
+use Illuminate\Support\Arr;
+
 MyBB\app('config')->set(
-	array_dot($config)
+	Arr::dot($config)
```

Comment:
I don't understand here. If we're using package `Illuminate\Support` here, why isn't it required from the `composer.json`?

Reply:
Probably because it's included with other dependencies as a required package for installing.

Reply:
Hmm, IMO it should be required as we use the library explicitly. Although it might not be, it's still possible for this lib to be dropped from various `Illuminate` lib dependencies. I may be wrong.